woocommerce-services.php

```diff
@@ -818,6 +818,7 @@ public function attach_hooks() {
 			add_action( 'woocommerce_email_after_order_table', array( $this, 'add_tracking_info_to_emails' ), 10, 3 );
 			add_filter( 'woocommerce_admin_reports', array( $this, 'reports_tabs' ) );
 			add_action( 'woocommerce_checkout_order_processed', array( $this, 'track_completed_order' ), 10, 3 );
+			add_action( 'admin_print_footer_scripts', [ $this, 'add_sift_js_tracker' ] );
```

Comment:
[WP coding standards array syntax](https://developer.wordpress.org/coding-standards/wordpress-coding-standards/php/#declaring-arrays):

```suggestion
			add_action( 'admin_print_footer_scripts', array( $this, 'add_sift_js_tracker' ) );
```

Reply:
Good catch, thanks!